gen/calc.py

```diff
@@ -412,6 +441,8 @@ def validate_bootstrap_tmp_dir(bootstrap_tmp_dir):
               }                                         \
             ]}',
         'dcos_remove_dockercfg_enable': "false",
+        'minuteman_min_named_ip': '11.0.0.0',
+        'minuteman_max_named_ip': '11.0.0.254',
```

Comment:
It should be okay to use `11.255.255.255`, since this isn't on a layer 2 segment.

The way that Minuteman works is by programming kernel flows (via IPTables conntrack).